CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.10)
+
+#project_name
+project(UniversityTimetable)
+
+#Specify the C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ggdb")
+
+#Include directories
+include_directories(include)
```

Comment:
Always give a full path, you can use CMAKE_SOURCE_DIR or other variables (do it in all places)